observability/src/main/java/io/grpc/observability/logging/Sink.java

```diff
@@ -29,6 +30,15 @@
    */
   void write(GrpcLogRecord message);
 
+  /**
+   * Writes the {@code message} to the destination.
+   * locationTgs to identify the source of logs
+   * customTags tags added by user
+   */
+  void write(GrpcLogRecord message,
+      Map<String, String> locationTags,
```

Comment:
Just realized that the locationTags and customTags are fixed/immutable once created by GlobalLoggingTags. What's the need then to pass everytime via write()? You can just set them in the GcpLogSink thru the constructor, right? Or is that not right?